scripts/move-contacts/laikipia-west.json

```diff
@@ -0,0 +1,457 @@
+[
```

Comment:
you probably need to include a csv / xlsx to json converter

Reply:
ya this was just quick and dirty. i wasn't planning to check this in less people say [here](https://medic.slack.com/archives/C066J5C8Z88/p1716338933731549) that they think it's useful. so far nobody said anything, so I'll just discard this